src/oci.c

```diff
@@ -372,8 +404,8 @@ cc_oci_kill (struct cc_oci_config *config,
 		return true;
 	}
 
-	/* stopping container */
-	config->state.status = OCI_STATUS_STOPPING;
+	/* signaling container, not all signals stop the container */
+	config->state.status = OCI_STATUS_SIGNALING;
```

Comment:
Nice, I always thought it was a bug, that we mark the container as stopping for any signal.
Is the string for OCI_STATUS_SIGNALING part of the OCI spec?